test/views/user_view_test.exs

```diff
@@ -38,7 +40,10 @@ defmodule CodeCorps.UserViewTest do
             ]
           },
           "slugged-route" => %{
-            "data" => nil
+            "data" => %{"id" => slugged_route.id |> Integer.to_string, "type" => "slugged-route"}
+          },
+          "stripe-customer" => %{
+            "data" => %{"id" => stripe_customer.id |> Integer.to_string, "type" => "stripe-customer"}
```

Comment:
I can see the logic in that, but do we actually risk confusing `ember-data` by doing so? If we're manually refreshing the user on authentication (we are), then it shouldn't matter.

Even the existence of those relationships leaking to other people seems problematic to me. Will think on this and ask around some.